js_modules/dagit/packages/core/src/workspace/asset-graph/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -175,9 +180,16 @@ export const LaunchAssetChoosePartitionsDialog: React.FC<{
         },
       });
 
-      handleLaunchResult(basePath, assetJobName, launchResult, {});
+      if (launchResult.data?.launchPipelineExecution.__typename === 'LaunchRunSuccess') {
+        showLaunchSuccessToast(history, launchResult.data.launchPipelineExecution);
+        setLaunching(false);
```

Comment:
since if & else both call setLaunching (false) you can move it up or down out of there if you want to, unless the exact order matters.